docs/stories/user_journey.md

```diff
@@ -0,0 +1,28 @@
+Hi Legends of Lightning Community!
+
+We've spent the last week working on a design to bring non-custodial financial products to mobile. The first use case we've focused on is "non-perpetual" CFD trading.
+If you are interested, we already have a productive, perpetual solution available called [ItchySats](https://www.itchysats.network/).
+
+Without further ado, here is the first user journey for creating a long position.
+
+![](../assets/2022-10-26_UserJourney-WalletCfdTrading.png)
+
+The journey starts after the user has already initialized her lightning wallet.
```

Comment:
Prefer using either Australian (British) English or American. You seem to use both (`initialized` and `analysed`) in short succession 
```suggestion
The journey starts after the user has already initialised her lightning wallet.
```